test/integration_tests/short/test_show_info_executor.py

```diff
@@ -128,6 +127,12 @@ def test_show_config_execution(self):
         with self.assertRaises(Exception):
             execute_query_fetch_all(self.evadb, "SHOW BADCONFIG")
 
+    def test_show_all_configs(self):
+        show_all_config_value = execute_query_fetch_all(self.evadb, "SHOW CONFIG")
+
+        # QUES : What do I test this against?
+        print(show_all_config_value)
```

Comment:
Is it possible to get all configs from a clean evadb instance and compare them? So this testcases also serve for default available config options.

Reply:
I can use the python dict that we created to check for those configs, but the values can be changed and we do not want it to fail if any change has been made. Maybe we only check for keys

Reply:
My opinion is that the test should fail when we add a new default configurations.  Or do you mean the other cases that will lead to the values to be changed?

Reply:
The problem is that values like the path are specific to each user and system like `EVADB_INSTALLATION_DIR`